coalib/coala.py

```diff
@@ -37,7 +37,8 @@ def main():
 
         console_printer = ConsolePrinter(print_colored=not args.no_color)
 
-        if args.json:  # needs to be checked in order to display bears in json
+        # needs to be checked in order to display bears in json
+        if args.json and not args.format:
```

Comment:
It shouldn't be possible that both are `True` here. Should already have been handled by `check_conflicts` before. So better `assert` here if you want to get sure...